scripts/globals/mobskills/frenzied_rage.lua

```diff
@@ -0,0 +1,27 @@
+---------------------------------------------
+-- Frenzied Rage
+--
+-- Description: Attack Boost
+-- Type: Enhancing
+-- Utsusemi/Blink absorb: N/A
+-- Range: Self
+-- Notes: 20% Attack Boost.
+---------------------------------------------
+require("scripts/globals/monstertpmoves")
+require("scripts/globals/settings")
+require("scripts/globals/status")
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    return 0
+end
+
+function onMobWeaponSkill(target, mob, skill)
+    local power = 20
```

Comment:
Oh, ok. I figured it was more of a "let the reader know" what the param was for instead of it being a magic number. But, I can nix them from now on... It will save some space. =)

Reply:
my line of thinking here is they are not required to be spelled out because anyone futzing with the function they are fed into should know that parameter/argument of the function is power just by looking at it:
https://github.com/DarkstarProject/darkstar/blob/master/scripts/globals/monstertpmoves.lua#L719
```lua
function MobBuffMove(mob, typeEffect, power, tick, duration)
```
(anytime we don't know what the functions arguments actually are, we should be looking at where the function is defined for that answer anyways)

if they are likely to be frequently edited (like everything in settings.lua), changed further along in the script, or would make a line to wide to read without excessive right/left scrolling, those are good indications it should be made a variable in my opinion.

Reply:
yes, you are correct! Roger that.